README.md

```
@@ -1,61 +1,137 @@
+### Tall
+- [ ] Create a controlled input component
+  - Implement input validation
+  - Handle user interactions
+- [ ] Build a sign-up form composed component
+  - Compose using the input components
+  - Implement form validation
+- [ ] Add Next button functionality
+  - Enable/disable based on validation
+  - Handle form submission
+
+### Grande
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)